design docs/durandal play by play.txt

```diff
@@ -0,0 +1,55 @@
+Gameplay Example
+
+Player1 joins the server and enters the customization screen.
+
+They choose a government entity with specializations:
+
+- Large Space Agency: Large budget, extensive existing responsibilities.
+- Old Space Agency: Experienced crews and engineers; slower adoption of new technologies and penalties when restructuring or significantly changing processes.
+- Crewed Missions Specialization: Easier and safer crewed missions but significantly increased difficulty and cost for autonomous missions.
+
+Entering the game, Player1 reviews their agency's current obligations: a schedule of commercial satellite launches and ISS-2 service missions. The budget overview indicates a surplus sufficient to consider new projects.
+
+Player1 browses available contractors and finds a rocket capable of carrying two of their regular payloads simultaneously. The contract offers cost efficiency but significantly reduces customization and oversight of safety standards, risking potential mission failures and lower reputation gains if accidents occur.
+
+After careful consideration, Player1 signs the contract. They experience reduced production overhead but retain significant overhead costs due to their intertwined internal structures.
+
+With commercial satellites bundled, launches are freed for a new project. Though a moon mining project is desirable, the budget remains insufficient. Player1 opts for an intermediate project: constructing additional modules for the ISS-2 to rent to corporations for increased revenue.
```

Comment:
Regarding budgeting limiting things, how does it limit their ability to perform a moon mining operation?